types/transaction.go

```diff
@@ -145,7 +157,7 @@ type TransactionWrapper struct {
 
 type TransactionV1 struct {
 	// Hex-encoded TransactionV1 hash
-	TransactionV1Hash *key.Hash `json:"hash"`
+	TransactionV1Hash key.Hash `json:"hash"`
```

Comment:
Hash, Header and Body instead of TransactionV1Hash, TransactionV1Header, TransactionV1Body?